en/docs/guides/user-mgt/enabling-notifications.md

```diff
@@ -0,0 +1,99 @@
+# Enabling Notifications for User Operations
+
+The primary objective of this is to send a notification to an external endpoint once a user operation has taken place. For example, you want to send an email to an email address once a new user is created or the password of an existing user is changed. This can be achieved by enabling notifications for user operations.
+
+The configuration differs based on the type of notifications you want to send to the external endpoint. The following instructions provide information on how to send an email when a user operation takes place.
+
+1. Enable the email sending configurations of the WSO2 Identity Server as explained [here](insert-configuring-email-sending-link). 
```

Comment:
there is already a page for email configurations called "configure-email-sending.md" in the fragments folder. we can directly link to that page or include it here